src/bindings/c/docs/how_to_wrap_openvino_interfaces_with_c.md

```diff
@@ -0,0 +1,81 @@
+# How to wrap OpenVINO interfaces with C
+
+Library `openvino_c` implements the most C++ interfaces provided in OpenVINO with C, the details mapping relationship can be found in [Mapping Relationship of Interfaces](./docs/mapping_relationship_of_interfaces.md). Here is the details about how to wrap interfaces(C++) from OpenVINO to interfaces(C), main steps including:
+ 1) define C interface name and input parameters
+ 2) check the validation of input parameters
+ 3) convert C input parameters to C++ interface using
+ 4) call C++ interface and get return
+ 5) convert C++ return object to C object
+
+```mermaid
+flowchart LR
+    subgraph c_wrap_process[The Flow about Wrapping C++ by C]
+        c_interface["Define C name"]
+        c_check_parameter["Input parameter checking"]
+        c_return_invalid["return ov_status_e::INVALID_C_PARAM"]
+
+        subgraph c_wrap_call_cpp[Try Call C++]
+            c_parameter_convert["convert C parameters to C++ parameters"]
+            c_call_cpp_interface["call C++ interface"]
+            cpp_parameter_convert["convert C++ returned objects to C object"]
+        end
+        
+        c_return_success["return ov_status_e::OK"]
+        c_return_exception["catch EXCEPTION and return"]
+
+        c_interface-->c_check_parameter
+
+        c_check_parameter--valid-->c_parameter_convert
+        c_check_parameter--invalid-->c_return_invalid
+
+        c_parameter_convert-->c_call_cpp_interface
+        c_call_cpp_interface-->cpp_parameter_convert
+
+        cpp_parameter_convert-->c_return_success
+        c_wrap_call_cpp-->c_return_exception
+    end
+```
+
+According to the ability, all C provided interfaces can be classified to three kinds of methods:
+- Wrap C++ interface to create object
+- Wrap C++ interface to operate object
+- C implement interfaces 
```

Comment:
kinda lost here does it mean Implement C interfaces or C-implemented interfaces?